lms/djangoapps/teams/static/teams/js/collections/base.js

```diff
@@ -27,7 +27,7 @@
                     var self = this,
                         deferred = $.Deferred();
                     if (force || this.isStale) {
-                        this.fetch()
+                        this.setPage(1)
```

Comment:
The sorting might have changed as a result of the collection updating -- that's what @andy-armstrong was talking about in earlier comments. In that case, we don't necessarily want to remain on the same page, since the topic may have shifted off the page due to re-sorting. After talking with @explorerleslie we decided to just reset to the first page as soon as the collection is changed.